src/main/java/seedu/address/logic/commands/BatchEditCommand.java

```diff
@@ -0,0 +1,117 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.PersonContainsTagsPredicate;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Edit all person with specified tag to change the specified tag with new tag.
+ */
+public class BatchEditCommand extends Command {
+    public static final String COMMAND_WORD = "batch-edit";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Change the specified tag with new tag"
+            + "Parameters:"
+            + PREFIX_TAG + "TAG" + " "
```

Comment:
I think the parameters here should be PREFIX_TAG + "OLDTAG " + PREFIX_TAG + "NEWTAG"
And the Example should be a real life example eg. PREFIX_TAG + "Sec1 " + PREFIX_TAG + "Sec 2 will change all tags named Sec1 to Sec 2"